PatientWebApplication/HealthClinic.DAL/Repository/RegistrationInPharmacyRepository.cs

```diff
@@ -29,7 +29,16 @@ public RegistrationInPharmacy getPharmacyApiKey(String apiKey)
         {
             return DbContext.Registrations.ToList().SingleOrDefault(registration => registration.ApiKey == apiKey);
         }
+        public Boolean Remove(String apiKey)
+        {
+            RegistrationInPharmacy registration2 = DbContext.Registrations.ToList().SingleOrDefault(registration => registration.ApiKey == apiKey);
+            try {
+                DbContext.Registrations.Remove(registration2);

```

Comment:
Write this in form:
"DbContext.Registrations.Remove( DbContext.Registrations.ToList().SingleOrDefault(registration => registration.ApiKey == apiKey));"